test_utils/factories.py

```diff
@@ -32,18 +32,20 @@ class Meta:
 
 
 # pylint: disable=no-member
-class BlacklistedCourseSkillFactory(factory.django.DjangoModelFactory):
+class CourseSkillsFactory(factory.django.DjangoModelFactory):
```

Comment:
We already have `CourseSkills` and `Skills` factory available in discovery, wouldn't it be redundant?

Reply:
I needed `CourseSkillsFactory` for tests inside taxonomy tests.